src/main/java/me/tomsdevsn/hetznercloud/objects/general/ErrorCode.java

```diff
@@ -0,0 +1,22 @@
+package me.tomsdevsn.hetznercloud.objects.general;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+public enum ErrorCode {
```

Comment:
Please also add the call specific error codes such as `caa_record_does_not_allow_ca` or `firewall_resource_not_found`.